### DISCUSSION TITLE
Feller EDIZIOdue Smart Light Control

### DISCUSSION THREAD

**Discussion by Pasci94** [Koenkk/zigbee2mqtt#9701] [Devices]
Does anybody have some experience with the Feller EDIZIOdue Smart Light Control?

**Comment by yvansandoz** [comment]
Did you find a way to connect your Feller EdizioDue with Zigbee2MQTT?

**Comment by UnconventionalMindset** [comment]
If anyone needs to know how, I managed with this guide: https://www.zigbee2mqtt.io/devices/PTM_215Z.html
